libs/optimizer/src/lib/scoringRules.ts

```diff
@@ -0,0 +1,111 @@
+import { scoringRules as xcScoreRules} from "igc-xc-score";
+
+export enum LeagueCode {
+  CZL,
+  CZE,
+  CZO,
+  FFVL,
+  LEO,
+  NOR,
+  UKC,
+  UKI,
+  UKN,
+  XContest,
+  XCPPG,
+  WXC
+}
+
+const scoringBaseModel = xcScoreRules['XContest'];
```

Comment:
We can not do that, because definitions in fact depends on functions in igc-xc-score that are not exported by this lib. we should in this case copy/paste this functions to make our own definitions.

Reply:
You can send a PR to the repo if it makes sense to export those functions or we can copy paste them here.